app/views/admin/attachments/_reference_fields.html.erb

```diff
@@ -3,7 +3,10 @@
   <p class="help-block">This determines the translations of the publication that the attachment will appear in.</p>
   <%= form.select :locale, options_for_locales(attachable.translated_locales), include_blank: 'All languages' %>
 <% end %>
-<%= form.text_field :isbn, label_text: 'ISBN' %>
+
+<%= form.text_field :isbn, label_text: "ISBN (web version)" %>
+<%= form.text_area :meta_data_contact_address, rows: 5, label_text: "Organisation's Contact Details" %>
```

Comment:
And indicate what, if any, styling options they have, because there's probably an expectation that any text box is a govspeak box (maybe not though - definitely check with content folk).